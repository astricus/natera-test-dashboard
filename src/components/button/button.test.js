import React from 'react';
import { shallow } from 'enzyme';
import Button from './button.component';

test('should render primary Button component', () => {
  expect(shallow(<Button primary={true}>Click me!</Button>)).toMatchSnapshot();
});
