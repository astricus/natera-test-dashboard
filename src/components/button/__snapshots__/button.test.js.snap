// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render primary Button component 1`] = `
<button
  className="btn btn--primary   "
  type="button"
>
  Click me!
</button>
`;
